pkg/providers/cf/cf_suite_test.go

```diff
@@ -1,4 +1,4 @@
-package cloud_foundry_test
+package cf_test
```

Comment:
Why do you think it's better to capitalize cloud foundry (`cf`) instead of using its full name (`cloud_foundry`)?

Reply:
Done